boards/openhwgroup/cv32a6_genesysII/doc/index.rst

```diff
@@ -0,0 +1,87 @@
+.. _cv32a6_genesys2:
+
+Digilent GenesysII with CV32A6 SoC
```

Comment:
The FPGA devboard in question seem to be called "Genesys 2", not "GenesysII": https://digilent.com/reference/programmable-logic/genesys-2/start

Though that makes me wonder - do we want to include the FPGA devboard name in the target name/documentation? Other FPGA-based targets don't seem to do that:
* https://docs.zephyrproject.org/latest/boards/enjoydigital/litex_vexriscv/doc/index.html
* https://docs.zephyrproject.org/latest/boards/others/neorv32/doc/index.html

But if we want to keep the name, I'd suggest you to change the target name from `cv32a6_genesysII` to `cv32a6_genesys_2` so that it has the name the vendor uses.

Reply:
I will change the spelling of the board name. 
I included it because I plan to send at least one additional PR with a slightly different configuration on a different development board (ARTY A7), as soon as the hardware configuration is finalized. The main differences between the configuration come down to the actual development board hardware, i.e., installed RAM and Ethernet PHY, so I wanted to be clear which zephyr board corresponds to which development board.
CVA6 also has support for additional development boards that I cannot test builds on, so there might be additional PRs in the future as well:
- Digilent Nexys Video
- Xilinx KC705
- Xilinx VC707